tests/pyreverse/functional/package_diagrams/type_check_imports/packages.mmd

```diff
@@ -0,0 +1,14 @@
+classDiagram
+  class type_check_imports {
+  }
+  class mod_a {
+  }
+  class mod_b {
+  }
+  class mod_c {
+  }
+  class mod_d {
+  }
+  mod_b --> mod_a
+  mod_d --> mod_a
+  mod_c -.-> mod_a
```

Comment:
This is not valid mermaid.

it should be:
`mod_c ..> mod_a`
as documented here:
https://mermaid.js.org/syntax/classDiagram.html#defining-relationship

github renders mermaid files, and if we try to open this one we get an error, see:
https://github.com/pylint-dev/pylint/blob/ffe1e714a3cfe83b5a627ade6d723489f2e10621/tests/pyreverse/functional/package_diagrams/type_check_imports/packages.mmd